src/EFCore.PG/Query/Expressions/Internal/PostgresFunctionExpression.cs

```diff
@@ -8,7 +8,7 @@
 using Microsoft.EntityFrameworkCore.Storage;
 using Npgsql.EntityFrameworkCore.PostgreSQL.Utilities;
 
-#nullable enable
+#pragma warning disable 8632
```

Comment:
😢

Reply:
Don't worry, it's temporary :) Once EF Core itself is fully annotated, I'll come back here and do the same. I just don't want to deal with a partially-annotated situation.